source/loader/layers/sanitizer/asan_interceptor.cpp

```diff
@@ -13,6 +13,7 @@
 
 #include "asan_interceptor.hpp"
 #include "ur_sanitizer_layer.hpp"
+#include <cxxabi.h>
```

Comment:
IMO, it'd better to extract these codes as a function, which is defined in `unified-runtime/source/loader/layers/sanitizer/linux/san_utils.cpp`, as it's specific to Linux